test/property.test.js

```diff
@@ -62,6 +62,121 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('should create model containing number type', function(done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'age',
+      type: 'number',
+      required: 'true',
+      defaultValue: '555555555555555555555.5'
+    });
+
+    propertyGenerator.run(function() {
+      var definition = common.readJsonSync('common/models/car.json');
+      var props = definition.properties || {};
+      expect(props).to.have.property('age');
+      expect(props.age).to.eql({
+        type: 'number',
+        required: true,
+        default: 555555555555555555555.5
+      });
+      done();
+    });
+  });
+
+  it('should create model containing array type', function(done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'options',
+      type: 'array',
+      required: 'true',
+      defaultValue: 'AWD,3.2L, navigation'
+    });
+
+    propertyGenerator.run(function() {
+      var definition = common.readJsonSync('common/models/car.json');
+      var props = definition.properties || {};
+      expect(props).to.have.property('options');
+      expect(props.options).to.eql({
+        type: 'array',
+        required: true,
+        default: ['AWD','3.2L','navigation']
+      });
+      done();
+    });
+  });
+
+  it('should create model containing date type', function(done) {
```

Comment:
If we are going to support both JS types of `Number` and `String` for the default value, we need to test both cases.